### PR TITLE
Store parsing fix

### PR DESCRIPTION
**Pull request by kripken**
It was broken.

We also didn't write stores properly either!

### CODE REVIEW

**Comment by kripken** [comment]
Anyone want to review?

File: src/s2wasm.h [hunk 1]
```diff
@@ -669,7 +669,7 @@ class S2WasmBuilder {
       skipComma();
       auto curr = allocator.alloc<Store>();
       curr->type = type;
-      int32_t bytes = getInt();
+      int32_t bytes = getInt()/8;
```
Comment (jfbastien) [inline src/s2wasm.h]: I'd use `CHAR_BIT` as the non-magic-number for "bits per byte".

**Comment by jfbastien** [comment]
One comment, looks good otherwise.

File: src/s2wasm.h (hunk 1 above)
Comment (kripken) [inline src/s2wasm.h]: that's the bits per character, isn't it? on some systems the bits per character might not be 8?

Comment (jfbastien) [inline src/s2wasm.h]: A `char` is always one byte in C. `CHAR_BIT` is "number of bits for smallest object that is not a bit-field (byte)". In C++ "For any object (other than a base-class subobject) of trivially copyable type T, whether or not the object holds a valid value of type T, the underlying bytes (1.7) making up the object can be copied into an array of char or unsigned char. If the content of the array of char or unsigned char is copied back into the object, the object shall subsequently hold its original value."

**Comment by kripken** [comment]
Ok, updated for that comment, and merged.

**Pull request merged**